src/cpp/session/modules/SessionCopilot.cpp

```diff
@@ -177,18 +186,81 @@ bool s_isSessionShuttingDown = false;
 // Project-specific Copilot options.
 projects::RProjectCopilotOptions s_copilotProjectOptions;
 
+FilePath copilotAgentPath()
+{
+   // Check for configured copilot path.
+   FilePath copilotPath = session::options().copilotPath();
+   if (copilotPath.exists())
+      return copilotPath;
+
+   using namespace core::system::xdg;
+
+#ifdef _WIN32
+   // on Windows, the copilot agent was previously downloaded to a different location.
+   // Delete and remove the old location after sufficient time has passed, in a future
+   // RStudio release.
+   FilePath oldAgentLocation = oldUserCacheDir().completeChildPath("copilot");
+   FilePath newAgentLocation = userCacheDir().completeChildPath("copilot");
+   if (oldAgentLocation.exists() && !newAgentLocation.exists())
+   {
+      Error error = newAgentLocation.copyDirectoryRecursive(newAgentLocation);
```

Comment:
Is this supposed to be copying the old location to the new location?

Reply:
Yes 😞  Thanks for catching that.